frigate/const.py

```diff
@@ -12,6 +12,13 @@
 PLUS_API_HOST = "https://api.frigate.video"
 BTBN_PATH = "/usr/lib/btbn-ffmpeg"
 
+# Audio Consts
+
+AUDIO_DETECTOR = "audio-detector"
+AUDIO_DURATION = 0.975
+AUDIO_FORMAT = "s16le"
+AUDIO_SAMPLE_RATE = 16000
+
 # Regex Consts
 
 REGEX_CAMERA_NAME = r"^[a-zA-Z0-9_-]+$"
```

Comment:
`REGEX_CAMERA_NAME = r"^[\w-]+$"`

Reply:
That has nothing to do with this PR so should be handled outside of it

Reply:
also in the getting started docs

![Screen Shot 2023-06-19 at 12 25 45 PM](https://github.com/blakeblackshear/frigate/assets/14866235/40801ea9-c4fb-4b4d-9330-f973db42371b)

Reply:
> also in the getting started docs

Wrong thread, by you absolutely right. it's my mistake